src/client/js/nav.js

```diff
@@ -18,7 +18,6 @@ function handleEvent (e) {
       nav.toggleAttribute('hidden')
       break
     case e.target.matches('.nav-item'):
-      e.target.style.setProperty('color', 'var(--purple-70)')
```

Comment:
I'm not sure why we wanted to set the colour of a link to purple when you click it, but now that the hover style has a purple background, that would result in purple-in-purple, so I removed it.

Reply:
Links should not only be distinguishable by colour so that people who have trouble distinguishing similar colours are able to recognise them. Additionally, the background colour makes it easier to recognise the clickable area, which helps users with motor impairments.

The Figma indeed did not include designs for the hover styles, so similar to when it does not include responsive designs, I came up with something myself. I will call it out in the desk check.

Reply:
Sounds good.  We remove the color and highlight in order to provide the user immediate feedback for the seconds before the new page load.